geometry_msgs/Point target
float64 targetRotation
geometry_msgs/Point dest
float64 destRotation
